ibtk/src/boundary/cf_interface/CartSideDoubleQuadraticCFInterpolation.cpp

```diff
@@ -163,11 +163,7 @@ static const int GHOST_WIDTH_TO_FILL = 1;
 /////////////////////////////// PUBLIC ///////////////////////////////////////
 
 CartSideDoubleQuadraticCFInterpolation::CartSideDoubleQuadraticCFInterpolation()
-    : d_patch_data_indices(),
-      d_consistent_type_2_bdry(false),
-      d_refine_op(new CartesianSideDoubleConservativeLinearRefine<NDIM>()),
-      d_hierarchy(nullptr),
-      d_cf_boundary(),
+    : d_refine_op(new CartesianSideDoubleConservativeLinearRefine<NDIM>()),
```

Comment:
Looks like clang-tidy isn't moving things that initialize `Pointer<T>` via `new T`?

Reply:
It didn't move most of these things :). I think it only moves built in types.

I didn't change these because I was getting an error that I thought was due to typecasting that couldn't be done in header files, but now I realize that it's just due to namespaces. I'll update these.